### DISCUSSION TITLE
Inventree with LDAP

### DISCUSSION THREAD

**Discussion by kimizumi1** [inventree/InvenTree#5450] [General]
Has anyone got InvenTree work with LDAP.
I have a hard time getting it to work. Either the whole InvenTree crashes or I get "No module named 'django_auth_ldap'" in Django logs

All needed python packages are installed. For some reason I can't get past this error

**Comment by SchrodingersGat** [comment]
@kimizumi1 you might want to refer to https://github.com/inventree/InvenTree/issues/3746

**Reply by wolflu05** [reply]
And also #5226 , is already ready from my pov except the installer stuff @matmair wanted to add before merge.
